src/client/game.cpp

```diff
@@ -3193,18 +3188,17 @@ void Game::handlePointingAtNode(const PointedThing &pointed,
 		}
 	}
 
-	if ((input->getRightClicked() ||
-			runData.repeat_rightclick_timer >= m_repeat_right_click_time) &&
+	if ((wasKeyDown(KeyType::PLACE) ||
+			runData.repeat_place_timer >= m_repeat_place_time) &&
 			client->checkPrivilege("interact")) {
-		runData.repeat_rightclick_timer = 0;
-		infostream << "Ground right-clicked" << std::endl;
+		runData.repeat_place_timer = 0;
+		infostream << "Place button pressed while looking at ground" << std::endl;
 
 		if (meta && !meta->getString("formspec").empty() && !random_input
 				&& !isKeyDown(KeyType::SNEAK)) {
-			// Report right click to server
-			if (nodedef_manager->get(map.getNode(nodepos)).rightclickable) {
+			// Report place action to server
+			if (nodedef_manager->get(map.getNode(nodepos)).can_place_onto)
 				client->interact(INTERACT_PLACE, pointed);
```

Comment:
The interact has to be sent to trigger the `on_rightclick` callback. This is not placement. (Well, the action is indeed called "place".)
Anyway, the `can_place_onto` is wrong. It has to be `rightclickable`, as in "has an `on_rightclick` callback".

Reply:
I'll just rename it back to `on_rightclick`. I don't want to stall this PR with the ongoing naming contest.